### PR TITLE
add hw15

### PR DESCRIPTION
**Pull request by bater091**

### CODE REVIEW

**Comment by bater091** [comment]
add hw15

File: src/main/java/com/levchenko/hw15/BoxVolumeComparator.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            return -1;
+        } else if (a.getVolume() < b.getVolume()) {
+            return 1;
+        } else {
+            return 1;
+        }
+    }
+
+
+}
```
Comment (Fabolo12) [inline src/main/java/com/levchenko/hw15/BoxVolumeComparator.java:20]: Sorts not by task condition

File: src/main/java/com/levchenko/hw15/MyList.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            if (temp == head && temp.getValue() != value) {
+                return null;
+
+            }
+            temp = temp.getPrev();
+
+
+        }
+        return temp.getValue();
+    }
```
Comment (Fabolo12) [inline src/main/java/com/levchenko/hw15/MyList.java:45]: Doesn't work if the list is empty

**Review by Fabolo12 — CHANGES_REQUESTED** [review]

**Review by Fabolo12 — APPROVED** [review]
10

**Pull request merged**